README.md

```diff
@@ -42,263 +33,7 @@ If the `pip install` command above gives a permissions error, install as a non-r
 pip install --user neurodocker
```

Comment:
it might be useful to add minimal python requiremnts

Reply:
Yes good idea. I believe it is 3.7+. Haven't tested 3.6 though.